### PR TITLE
Tutorial: Install TYPO3 on DDEV

### PR DESCRIPTION
**Pull request by linawolf**
releases: main, 11.5

### CODE REVIEW

File: Documentation/Installation/Index.rst [hunk 1]
```diff
@@ -23,6 +23,19 @@ Installation
...
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header h3
+
+            .. rubric:: :ref:`Tutorial: Locally install TYPO3 with ddev <installation-ddev-tutorial>`
+
+         .. container:: card-body
+
+            We show you how to install TYPO3 on a local machine utilizing
+            DDev, Docker and Composer.
```
Comment (brotkrueml) [inline Documentation/Installation/Index.rst:37]:
```suggestion
            DDEV, Docker and Composer.
```

As it writes itself: https://ddev.readthedocs.io/en/stable/

File: Documentation/Installation/TutorialDdev.rst [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+        ..  code-block:: bash
+
+            ddev config
+
+            # Give the following answers when prompted:
+
+            Project name (t3example):
+
+            Docroot Location (current directory):public
```
Comment (brotkrueml) [inline Documentation/Installation/TutorialDdev.rst:67]:
```suggestion
            Docroot Location (current directory): public
```

File: Documentation/Installation/TutorialDdev.rst [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    .. rst-class:: bignums-warning
+
+    a.  You can do this from console:
+
+        ..  code-block:: bash
+
+            ddev typo3cms install:setup
+
+        When prompted enter a username and password for you main admin user
+        (who is also system maintainer).
```
Comment (brotkrueml) [inline Documentation/Installation/TutorialDdev.rst:121]: Maybe link "system maintainer", not everyone may know, what that is: https://docs.typo3.org/m/typo3/tutorial-getting-started/11.5/en-us/UserManagement/BackendPrivileges/Index.html#system-maintainers

File: Documentation/Installation/TutorialDdev.rst [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+to fake sending mails. You can see what mails have been send here:
+
+..  code-block:: bash
+
+    ddev launch -m
+
+How do I get rid of that project now?
+=====================================
+
+If you want to all projects from running on DDEV you can call:
```
Comment (brotkrueml) [inline Documentation/Installation/TutorialDdev.rst:178]: This sentence sounds strange. Maybe: "If you want to stop all DDEV projects ..."

File: Documentation/Installation/TutorialDdev.rst [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+
+..  code-block:: bash
+
+    ddev poweroff
+
+The projects will stay configured and the database persisted.
+
+If you want to delete the project you just deleted you can delete it by
```
Comment (brotkrueml) [inline Documentation/Installation/TutorialDdev.rst:186]:
```suggestion
If you want to delete the project you just created you can delete it by
```
A little bit "delete" too often.

File: Documentation/Installation/TutorialDdev.rst [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+calling the following in your new projects root folder:
```
Comment (brotkrueml) [inline Documentation/Installation/TutorialDdev.rst:187]:
```suggestion
calling the following command in your new projects root folder:
```

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: Documentation/Installation/TutorialDdev.rst [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+to fake sending mails. You can see what mails have been send here:
+
+..  code-block:: bash
+
+    ddev launch -m
+
+How do I get rid of that project now?
+=====================================
+
+If you want to all projects from running on DDEV you can call:
```
Comment (linawolf) [inline Documentation/Installation/TutorialDdev.rst:178]:
```suggestion
If you want to stop all projects from running on DDEV you can call:
```

File: Documentation/Installation/TutorialDdev.rst [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+    .. rst-class:: bignums-warning
+
+    a.  You can do this from console:
+
+        ..  code-block:: bash
+
+            ddev typo3cms install:setup
+
+        When prompted enter a username and password for you main admin user
+        (who is also system maintainer).
```
Comment (linawolf) [inline Documentation/Installation/TutorialDdev.rst:121]:
```suggestion
        (who is also :ref:`system maintainer <system-maintainer>`).
```

File: Documentation/Installation/TutorialDdev.rst [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+
+We show you step-by-step how to install TYPO3 on a local machine utilizing
+DDEV, Docker and Composer.
+
+..  youtube:: HW7J3G1SqZw
+
+.. contents::
+   :local:
+
+Preparations to install DDev
```
Comment (brotkrueml) [inline Documentation/Installation/TutorialDdev.rst:19]:
```suggestion
Preparations to install DDEV
```

File: Documentation/Installation/TutorialDdev.rst [hunk 10]
```diff
@@ -0,0 +1,196 @@
...
+        (who is also :ref:`system maintainer <system-maintainer>`).
+
+        And enter a name for the site that will be displayed in the backend.
+
+        You can set up a very basic site :bash:`site` displaying "Hello World"
+        or just do nothing :bash:`no`.
+
+    b.  Or via GUI:
+
+        #.  Create a file called FIRST_INSTALL in your webroot
```
Comment (brotkrueml) [inline Documentation/Installation/TutorialDdev.rst:130]:
```suggestion
        #.  Create a file called :file:`FIRST_INSTALL` in your webroot
```

**Review by brotkrueml — APPROVED** [review]

**Comment by linawolf** [comment]
Let us wait for the review of @tomwarwick please

File: Documentation/Installation/TutorialDdev.rst [hunk 11]
```diff
@@ -0,0 +1,181 @@
...
+During the installation setup process TYPO3 created all the tables it needed. If you want to have a look at the database, you can run the following command:
+
+..  code-block:: bash
+
+    ddev launch -p
+
+Sending E-Mail
+--------------
+
+DDEV modified :file:`public/typo3conf/AdditionalConfiguration.php`
```
Comment (linawolf) [inline Documentation/Installation/TutorialDdev.rst:151]: Actually DDEV creates this file. If there would have been an AdditionalConfiguration already, DDEV would not have modified it

File: Documentation/Installation/TutorialDdev.rst [hunk 12]
```diff
@@ -0,0 +1,181 @@
...
+Stopping a DDEV Instance
+------------------------
+
+If you want to stop all projects from running you can call:
+
+..  code-block:: bash
+
+    ddev poweroff
+
+The projects will stay configured and the database will be persisted.
```
Comment (linawolf) [inline Documentation/Installation/TutorialDdev.rst:167]: shall we use plural databases? Each project has its own database

File: Documentation/Installation/TutorialDdev.rst [hunk 13]
```diff
@@ -0,0 +1,181 @@
...
+
+Deleting a DDEV Instance
+------------------------
+
+If you want to delete the project you just created you can delete it by
```
Comment (linawolf) [inline Documentation/Installation/TutorialDdev.rst:172]: delete.. delete... the sentence sounds strange...

File: Documentation/Installation/TutorialDdev.rst [hunk 14]
```diff
@@ -0,0 +1,181 @@
...
+
+
+Managing the Database
+---------------------
+
+Upon calling :bash:`ddev config` DDEV automatically created a database for
+you. DDEV also created a file called :file:`public/typo3conf/AdditionalConfiguration.php`
+in which it stored the database credentials for you.
+
+During the installation setup process TYPO3 created all the tables it needed. If you want to have a look at the database, you can run the following command:
```
Comment (linawolf) [inline Documentation/Installation/TutorialDdev.rst:142]: Line to long

**Review by tomwarwick — APPROVED** [review]

**Pull request merged**